book/pose.html

```diff
@@ -1421,12 +1421,15 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
       <ol type="a">
         <li> What are the decision variables in this optimization problem? Be
         specific. </li>
-        <li> How do the decision variables show up in the constraints?</li>
         <li>What is the value of the objective function for $p^O = (0,0)$? What
         about (3,10)? And (6,12)?</li>
-        <li>If you plotted the objective function and constraints as a function
+        <li>If you plotted the objective function as a function
         of your decision variables, what shape would it be? Explain the
         significance of this.</li>
+       <li> Consider the general case where rotation is fixed with an equal number of scene and model points. 
+        We only need to find the translational component. Solving this optimization problem,
+        what quantity do we find for each decision variable? 
+        (HINT: We want to minimize the squared distance of all the transformed scene points compared to their corresponding model points.) </li>
```

Comment:
I like the new part! Maybe we should add a comment like: "Your solution should be correct even when the model and scene points cannot be perfectly aligned." to avoid solutions that assume perfect alignment, and thus just take an arbitrary pair of scene and model points and take the translation

Reply:
Added!